privacyidea/lib/tokens/radiustoken.py

```diff
@@ -215,7 +469,8 @@ def check_otp(self, otpval, counter=None, window=None, options=None):
             radius_secret = binascii.unhexlify(secret.getKey())
 
         # here we also need to check for radius.user
-        log.debug("checking OTP len:{0!s} on radius server: {1!s}, user: {2!r}".format(len(otpval), radius_server, radius_user))
+        log.info("checking OTP len:{0!s} on radius server: {1!s}, user: {2!r}".format(len(otpval), radius_server,
```

Comment:
In most cases we only have ``debug`` logs on the token library level.
I would prefer to keep it that way.
If special logs are needed from certain modules, it is always possible to create a more sophisticated python log config and log debug only from certain modules.

Reply:
This was one I failed to switch back to debug.